TicTacToeApp/GamePrompt.swift

```diff
@@ -0,0 +1,29 @@
+
+public class GamePrompt {
+
+  public init() {
+
+  }
+
+  public func promptFor(game: Game) -> String {
+    if game.playerWonLastTurn(game.getInactivePlayer()) {
+      return playerWinPrompt(game.getInactivePlayer())
+    } else if game.isADraw() {
+      return "Players tied in a draw."
```

Comment:
I would extract this to a private function. Right now, the three different if branches have a different level of abstraction. i.e. The other two branches call a function, whereas this branch just returns a string.